examples/user_guide/Custom_Components.ipynb

```diff
@@ -153,7 +153,9 @@
     "  <option id=\"option\">${obj}</option>\n",
     "  {% endfor %}\n",
     "</select>\n",
-    "```"
+    "```\n",
+    "\n",
+    "Here we wrap the `${obj}` in the `<option id=\"option\">` element. But depending on your use case, you can use other tag on the form `<name-of-tag id=\"option\">`."
```

Comment:
The important thing to note here is that the wrapping element **must** have an `id`. It's also worth noting that the only reason to use an explicit loop (rather that just inserting `${children_parameter}`) is to be able to wrap each component in a custom tag.